README.md

```diff
@@ -1,5 +1,49 @@
-# $${\color{red}UNDER---CONSTRUCTION}$$
-This repository is currently under construction, but will provide Jupyter Notebooks that demonstrate how to use the JWST Calibration Pipeline
+# JWST Pipeline Notebooks
 
-We are accepting internal pull requests, but only performing basic checks at the moment for notebooks, and not running the full execution workflow.
+![STScI Logo](_static/stsci_header.png)
 
+> [!IMPORTANT]
+> JWST requires a C compiler for dependencies and is currently limited to Python 3.10, 3.11, or 3.12.
+
+> [!NOTE]
+> Linux and MacOS platforms are tested and supported.  Windows is not currently supported.
+
+The ``jwst_pipeline_notebooks`` repository contains python-based Jupyter notebooks that illustrate how to process JWST data through the STScI science calibration pipeline (``jwst``;  [https://github.com/spacetelescope/jwst](https://github.com/spacetelescope/jwst)).  An overview of the pipeline can be found at [https://jwst-docs.stsci.edu/jwst-science-calibration-pipeline](https://jwst-docs.stsci.edu/jwst-science-calibration-pipeline).
+
+Notebooks are organized according to instrument and observing mode.  Each notebook is designed to process data from uncalibrated raw FITS files to end-stage Level 3 data products (calibrated imaging mosaics, 3-D data cubes, 1-D extracted spectra, etc.).  These notebooks by default run in 'demo' mode, for which they will download and process example data drawn from the [MAST archive](https://archive.stsci.edu/).  They are, however, designed to be simple to run on arbitrary local data sets as well by configuring input directories accordingly.
+
+These notebooks are modular, allowing users to enable or disable different stages of processing.  Likewise, they provide examples of how to customize pipeline processing for specific science cases. 
+
+The following table summarizes the notebooks currently available:
+
+| Instrument | Observing Mode | JWST Build | Minimum ``jwst`` version | Notes                                         |
+|------------|----------------|------------|--------------------------|-----------------------------------------------|
+| MIRI       | MRS            | 11.0       | 1.15.1                   |                                               |
+| NIRISS     | Imaging        | 11.0       | 1.15.1                   |                                               |
+
+While each notebook has been constructed for use with a given minimum version of the ``jwst`` pipeline software, they are generally compatible with more-recent versions (see the JWST pipeline [version history](https://jwst-docs.stsci.edu/jwst-science-calibration-pipeline/jwst-operations-pipeline-build-information)).
```

Comment:
"generally" compatible? Do we want to be definitive here? For example, "The following table summarizes the notebooks currently available, and the versions they are compatible with" .  Then keep this table up to date?

Reply:
Fair point, and related in general to how often the notebooks will be updated.  For now I've removed this statement in favor of just stating a version.  Maybe a minimum and a maximum version number rather than listing each one though?

Reply:
I'm inclined not to, as jwst itself will pin the appropriate context and I'd worry that this table could drift out of sync.

Reply:
Even in the current interim mode where the crds client and service is still being updated betwen 1.14 and 1.16? The readme might want to minimally mention the proper use of the virtual contexts and point folks away from setting CRDS_CONTEXT to an explicit pmap  (this should be updated in the notebook text as well)

Reply:
I do agree with Megan that one environment per notebook is the best.

  I don't see in your instructions is that users could change the version of the jwst cal pipeline in the requirements.txt file. I hope soon we will get the CRDS software that will match the context with version so the user will not have to worry about it. Though, we need to clarify dependency on the context. Maybe this you want to put in JDox an add a link?

Reply:
Just updated the readme to point to the now-available JDox pages.  It briefly describes how running the notebook will automatically select the best context, but to see JDox for information about choosing non-default contexts.

For beginning users this should be enough- any new installs from this repo will automatically get this behaviour.

Reply:
Resolving this conversation, as I think it's been dealt with per @sosey 's comments below.